ush/templates/FV3LAM_wflow.xml

```diff
@@ -403,19 +423,111 @@ MODULES_RUN_TASK_FP script.
+************************************************************************
+-->
+
+  <task name="&ANAL_GSI_RESTART_TN;" cycledefs="dacycle" maxtries="{{ maxtries_anal_gsi }}">
+
+    &RSRV_ANALYSIS;
+    &WALL_LIMIT_ANAL;
+
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_ANAL_TN;" "&JOBSDIR;/JREGIONAL_RUN_ANAL"</command>
+    {% if machine in ["JET", "HERA"] -%}
+    <cores>{{ ncores_run_anal }}</cores>
+    <native>{{ native_run_anal }}</native>
+    {% else -%}
+    <nodes>{{ nnodes_run_anal }}:ppn={{ ppn_run_anal }}</nodes>
+    <nodesize>&NCORES_PER_NODE;</nodesize>
+    {% endif -%}
+    <walltime>{{ wtime_run_anal }}</walltime>
+    <jobname>&ANAL_GSI_RESTART_TN;</jobname>
+    <join><cyclestr>&LOGDIR;/&ANAL_GSI_RESTART_TN;_@Y@m@d@H.log</cyclestr></join>
+
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+    <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_ROOT</name><value><cyclestr>&CYCLE_BASEDIR;</cyclestr></value></envar>
+    <envar><name>BKTYPE</name><value><cyclestr>0</cyclestr></value></envar>
+
+    <dependency>
+      <or>
+        <and>
+          <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
+          <taskdep task="&MAKE_LBCS_TN;"/>
+          <datadep age="00:00:05:00"><cyclestr offset="-{{ da_cycle_interval_hrs }}:00:00">&CYCLE_BASEDIR;/@Y@m@d@H/dynf00{{ da_cycle_interval_hrs }}.nc</cyclestr></datadep>
+        </and>
+        <and>
+          <taskdep task="&MAKE_LBCS_TN;"/>
+          <timedep><cyclestr offset="&START_TIME_ANALYSIS;">@Y@m@d@H@M00</cyclestr></timedep>
+          <or>
+            {%- for h in range(da_cycle_interval_hrs+da_cycle_interval_hrs, 6+1, da_cycle_interval_hrs) %}
+            <datadep age="00:00:05:00"><cyclestr offset="-{{ h }}:00:00">&CYCLE_BASEDIR;/@Y@m@d@H/dynf{{ "%03d" % h}}.nc</cyclestr></datadep>
+            {%- endfor %}
+          </or>
+        </and>
+      </or>
+    </dependency>
+
+  </task>
+
+{% endif -%}
 <!--
 ************************************************************************
 ************************************************************************
 -->
-  <task name="&RUN_FCST_TN;{{ uscore_ensmem_name }}" cycledefs="forecast" maxtries="{{ maxtries_run_fcst }}">
+  <task name="&RUN_FCST_TN;{{ uscore_ensmem_name }}" cycledefs="forecast,dacycle" maxtries="{{ maxtries_run_fcst }}">
 
     &RSRV_FCST;
     &WALL_LIMIT_FCST;
 
     <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_FCST_TN;" "&JOBSDIR;/JREGIONAL_RUN_FCST"</command>
   {% if machine in ["JET", "HERA"]  %}
     <cores>{{ ncores_run_fcst }}</cores>
-    <native>{{ native_run_fcst }} &RRFS_RESERVE;</native>
+    <native>{{ native_run_fcst }}</native>
```

Comment:
This should include the reservation variable as in RRFS_dev1 branch.

Reply:
Two native setups work in my test and is used in current dev2. 
Do you know this limitation of native is old requirement or a new one?

Reply:
It's in the documentation. Please leave as it is in dev1.

Reply:
OK